src/filters/display-name-filter.js

```diff
@@ -0,0 +1,26 @@
+(function (angular, _) {
+    'use strict';
+
+    var drf = angular.module('drf-display-name-filter', []);
+
+    // A filter to return a display name.
```

Comment:
Maybe a bit more description here?
I.e. it matches the input slug in an array of choices, and returns the matching display_name for that slug